dockers/docker-orchagent/orchagent.sh

```diff
@@ -121,6 +121,12 @@ if [[ x"${ORCHDAEMON_RING_ENABLED}" == x"true" ]]; then
     ORCHAGENT_ARGS+=" -R"
 fi
 
+# Add heartbeat interval when enabled
+HEARTBEAT_INTERVAL=`sonic-db-cli CONFIG_DB hget  "HEARTBEAT|orchagent" "heartbeat_interval"`
+if [ ! -z "$HEARTBEAT_INTERVAL" ] && [ $HEARTBEAT_INTERVAL != "null" ]; then
+    ORCHAGENT_ARGS+=" -I $HEARTBEAT_INTERVAL"
```

Comment:
>$HEARTBEAT_INTERVAL [](http://example.com/codeflow?start=25&length=19)

Possible to have vuln of argument injection? like HEARTBEAT_INTERVAL="1; malicous_command" ?